.env

```diff
@@ -266,3 +266,11 @@ CURL_DB_URL=https://raw.githubusercontent.com/wiki/drevops/drevops/db_d8.dist.sq
 ##### DATABASE_IMAGE=drevops/drevops-mariadb-drupal-data-demo-8.x
 #;> DATABASE_IMAGE
 #;> DEMO
+
+# Development settings
+DREVOPS_LOCAL_REPO=1
```

Comment:
Setup for my local environment to enforce installing via local repository rather than from drevops remote.